### ISSUE TITLE
frequent crashes when connecting

### ISSUE THREAD

**Issue by shokifrend007** [Sanqui/irctr#3]
Hello,
when I try to connect it often crashes after entering my nickname.
Attached is a crash dump (just remove the `.txt`) and a video.
[crash_dump_00000000.dmp.txt](https://github.com/Sanqui/irctr/files/8353549/crash_dump_00000000.dmp.txt)
https://drive.google.com/file/d/1bqz6NiX8XkIVF2Kmy8JyjrQv6PNe7Och/view?usp=sharing

**Comment by Sanqui** [comment]
I'm gonna be honest, this was a proof of concept I made over five years ago and it's quite unlikely for me to develop it further at this time, but let me ask, are you actually using the client regularly? Would it be useful for you to see this fixed? 😮

**Comment by shokifrend007** [comment]
Yeah, you're right. I don't actually use irc. I just saw the opportunity to fix something.
